shell/common/node_bindings.cc

```diff
@@ -353,6 +353,14 @@ void SetNodeOptions(base::Environment* env) {
       "--max-http-header-size",
   });
 
+  if (env->HasVar("NODE_EXTRA_CA_CERTS")) {
+    if (!electron::fuses::IsNodeOptionsEnabled()) {
+      LOG(ERROR) << "The NODE_OPTIONS fuse must be enabled in order to use "
+                    "NODE_EXTRA_CA_CERTS";
+      env->SetVar("NODE_EXTRA_CA_CERTS", "");
```

Comment:
Hm, why do we UnSetVar above but SetVar("") here?

Reply:
@nornagon I matched what was there for `NODE_OPTIONS` in each location - however, we could likely unify them so that we're calling `UnsetVar` unless we're overwriting them unless I'm missing a historical reason.

Reply:
Done in 1cb2c7d5eb